resources/lang_chs.yml

```diff
@@ -0,0 +1,88 @@
+# Chinese Simplifyed Translation
+# Not Fully Checked
+# Authors:
+# Thunder33345
+
+#Join
+join-message: "&6此服务器需要密码认真."
+new-player: "&e欢迎 {player} 来到此服务器"
+
+#Login
+login-message: "&6请登入通过 /login <密码>\n&忘记您的密码? 用 /forgotpassword <pin> <新密码>"
+login-popup: "&6请登入."
+login-tip: "&6请登入."
```

Comment:
You are missing login-title, login-subtitle, & login-actionbar (Same as login-popup & login-tip).